### PR TITLE
Add cloud and snow probabilities

### LINKED ISSUE #128: Add cloud and snow probabilities
Cloud and Snow probabilities were added at some point the S2 processing.  Add them to assets:

CLD_20m.jp2, CLD_60m.jp2 - Cloud probabilities

SNW_20m.jp2, SNW_60m.jp2 - Snow probabilities

### PR DESCRIPTION
**Pull request by matthewhanson**
## Related issues

- Closes #128, and closes PR #119 includes DEFAULT_TOLERANCE change

## Description

Adds two new assets for cloud and snow probabilities

## Checklist

- [x] Includes tests
- [x] Includes documentation
- [x] Updates CHANGELOG

### CODE REVIEW

**Comment by philvarner** [comment]
This looks like it changes the default scale instead of the default tolerance?

File: tests/data-files/S2A_OPER_MSI_L2A_TL_VGS1_20220401T110010_A035382_T34LBP/expected_output.json [hunk 1]
```diff
@@ -1803,6 +1803,72 @@
         "reflectance"
       ]
     },
+    "cloud": {
```
Comment (philvarner) [inline tests/data-files/S2A_OPER_MSI_L2A_TL_VGS1_20220401T110010_A035382_T34LBP/expected_output.json:1806]: why were these added to only one of the many test files?

**Review by philvarner — CHANGES_REQUESTED** [review]
discrepancy between the PR description indicating a default tolerance change, and the default scale being changed.

**Comment by matthewhanson** [comment]
Ah, good catch @philvarner , fixed and pushed.

File: tests/data-files/S2A_OPER_MSI_L2A_TL_VGS1_20220401T110010_A035382_T34LBP/expected_output.json (hunk 1 above)
Comment (matthewhanson) [reply tests/data-files/S2A_OPER_MSI_L2A_TL_VGS1_20220401T110010_A035382_T34LBP/expected_output.json:1794]: Hmm, good question, I used https://github.com/stactools-packages/sentinel2/blob/main/scripts/create_expected.py to update the test files which does exclude some.  And then some of those tests are for the SAFE archives, and this only adds assets when using the s3 bucket.

**Review by gadomski — APPROVED** [review]

**Review by philvarner — APPROVED** [review]
